### PR TITLE
G2 t3 ingredient entity

### PR DESCRIPTION
**Pull request by jdselby24**
Created  class for ingredient entities and checkbox.

### CODE REVIEW

File: src/IngredientEntity.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace GRUB; 
+
+/* Class for IngredientEntities */
```
Comment (sudokufan) [inline src/IngredientEntity.php:5]: don't think you need this, pretty self explanatory

**Review by mgallagher56 — COMMENTED** [review]
Unit Test please

File: src/IngredientEntity.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    /**
+     * Generates HTML form checkbox for ingredient
+     *
+     * @return string HTML string
+     */
+    public function generateHTML(): string{
+        $html = "<div class='ingredient'>";
+        $html .= "<input type='checkbox' name=".$this->getName()."/>";
```
Comment (sudokufan) [inline src/IngredientEntity.php]:
```suggestion
        $html .= "<input type='checkbox' name=" . $this->getName() . "/>";
```
added space for clarity

File: src/IngredientEntity.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        $html .= "<p>".$this->getName()."</p>";
```
Comment (sudokufan) [inline src/IngredientEntity.php:36]:
```suggestion
        $html .= "<p>" . $this->getName() . "</p>";
```

**Review by sudokufan — COMMENTED** [review]
just cleanup suggestions

File: src/IngredientEntity.php (hunk 1 above)
Comment (jdselby24) [reply src/IngredientEntity.php:5]: Ashley said yes we do

File: src/IngredientEntity.php (hunk 2 above)
Comment (jdselby24) [reply src/IngredientEntity.php]: Its all one block, doesn't make sense to add space

File: src/IngredientEntity.php (hunk 3 above)
Comment (jdselby24) [reply src/IngredientEntity.php]: Its all one block, doesn't make sense to add space

File: src/IngredientEntity.php (hunk 1 above)
Comment (sudokufan) [reply src/IngredientEntity.php:5]: shut my mouth then

**Comment by sudokufan** [comment]
well done everybody

**Pull request merged**